content/general/concepts/what-is-graphql/what-is-graphql.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'What is GraphQL?' # This is the name that appears on the hub page for this language. Pay attention to capitalization and punctuation!
+Description: 'GraphQL is an open-source query language.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'APIs'
+  - 'Servers'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'paths/back-end-engineer-career-path'
+  - 'paths/front-end-engineer-career-path'
+---
+
+## History of GraphQL
+
+GraphQL was developed by Facebook in 2012 to address internal issues with their mobile applications. As an alternative to typical REST architecture, GraphQL became open-source in 2015. Consequently, GraphQL makes data transfers more efficient, network usage for Facebook's mobile applications was drastically reduced. As of now, GraphQL has attracted a large number of developers and is becoming increasingly popular for client-server communication.
+
+## What is GraphQL?
+
+GraphQL, an open-source query language as described in [GraphQL](graphql.org)'s official website. It allows developers to consume, construct, and create APIs in their own unique way.
```

Comment:
Let's remove the hyperlink here and replace it above (per my last comment). Also, 

> GraphQL, an open-source query language as described in GraphQL's official website.

I don't think this reads as a complete sentence. Could we, perhaps, incorporate the next sentence and rewrite it as "GraphQL is an open-source query language that allows developers to consume, construct, and create APIs in their own unique way".